Description: safer-buffer is not in debian and also not really needed, drop it
Author: Jérémy Lal <kapouer@melix.org>
Last-Update: 2019-01-06
Forwarded: not-needed
--- a/lib/iconv.js
+++ b/lib/iconv.js
@@ -20,7 +20,6 @@
 
 var stream = require('stream');
 var util = require('util');
-var Buffer = require('safer-buffer').Buffer;
 
 var bindings;
 try {
--- a/test/test-basic.js
+++ b/test/test-basic.js
@@ -18,7 +18,6 @@
 
 var assert = require('assert');
 var Iconv = require('../lib/iconv').Iconv;
-var Buffer = require('safer-buffer').Buffer;
 
 // unknown source/target encoding
 assert.throws(function() { new Iconv('utf-8', 'xxx'); });
--- a/test/test-big-buffer.js
+++ b/test/test-big-buffer.js
@@ -18,7 +18,6 @@
 
 var Iconv = require('../lib/iconv').Iconv;
 var assert = require('assert');
-var Buffer = require('safer-buffer').Buffer;
 
 var iconv = new Iconv('UTF-8', 'UTF-16LE');
 
--- a/test/test-stream.js
+++ b/test/test-stream.js
@@ -21,7 +21,6 @@
 var stream = require('stream');
 var net = require('net');
 var fs = require('fs');
-var Buffer = require('safer-buffer').Buffer;
 var PORT = 12345;
 
 assert(new Iconv('ascii', 'ascii') instanceof stream.Stream);
